src/server/typingsInstaller/nodeTypingsInstaller.ts

```diff
@@ -85,6 +85,11 @@ namespace ts.server.typingsInstaller {
                 throttleLimit,
                 log);
             this.npmPath = npmLocation !== undefined ? npmLocation : getDefaultNPMLocation(process.argv[0]);
+
+            // If the NPM path contains spaces and isn't wrapped in quotes, do so.
+            if (this.npmPath.indexOf(" ") !== -1 && this.npmPath[0] !== `"`) {

```

Comment:
This check is, current, a no-op if the default location was used, as currently that already wraps in quotes. It seems safer (and clearer to me) against any future changes though to just here always check if it needs wrapping, however it was obtained.